examples/barotropicqg_decaying_topography.jl

```diff
@@ -0,0 +1,233 @@
+# # Decaying barotropic QG turbulence over topography
+#
+#md # This example can be run online via [![](https://mybinder.org/badge_logo.svg)](@__BINDER_ROOT_URL__/generated/barotropicqg_decaying_topography.ipynb). 
+#md # Also, it can be viewed as a Jupyter notebook via [![](https://img.shields.io/badge/show-nbviewer-579ACA.svg)](@__NBVIEWER_ROOT_URL__/generated/barotropicqg_decaying_topography.ipynb).
+# 
+# An example of decaying barotropic quasi-geostrophic turbulence over topography.
+
+using FourierFlows, Plots, Printf, Random
+
+using Statistics: mean
+using FFTW: irfft
+
+import GeophysicalFlows.BarotropicQG
+import GeophysicalFlows.BarotropicQG: energy, enstrophy
+
+
+# ## Choosing a device: CPU or GPU
+
+dev = CPU()     # Device (CPU/GPU)
+nothing # hide
+
+
+# ## Numerical parameters and time-stepping parameters
+
+      n = 128            # 2D resolution = n²
+stepper = "FilteredRK4"  # timestepper
+     dt = 0.05           # timestep
+ nsteps = 2000           # total number of time-steps
+ nsubs  = 10             # number of time-steps for intermediate logging/plotting (nsteps must be multiple of nsubs)
+nothing # hide
+
+
+# ## Physical parameters
+
+L = 2π        # domain size
+nothing # hide
+
+# Define the topographic potential vorticity, ``\eta = f_0 h(x, y)/H``
```

Comment:
Maybe be more descriptive; e.g. Define the topographic potential vorticity, ``\eta = f_0 h(x, y)/H``. The topography here is an elliptical mound at (1,1), and an identically-shaped depression at (-1,-1) that has an opposing and smaller amplitude.